runtime/etc/Makefile.mli-shared

```diff
@@ -22,6 +22,9 @@
 
 include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.mli-common
 
+# Use the C++ compiler as our linker.
```

Comment:
This comment falls into the "obvious but not informative" category for me. Can you add a clause that says why it's necessary? Or even rephrase as "We can't just use the C compiler to link with, because <reason>."